### PR TITLE
adds .travis.yml + build badge in readme

### PR DESCRIPTION
**Pull request by mindreframer**
All that is left is to activate the travis.ci build at: 
https://travis-ci.org/henrik/ (works great with my account: https://travis-ci.org/mindreframer/ectoo) 

Cheers!

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+elixir:
+  - 1.1.0
+otp_release:
+  - 17.4
+addons:
+  postgresql: "9.4"
+sudo: false # to use faster container based build environment
+env:
+  - ELIXIR_ERL_OPTIONS=""
+  - ELIXIR_ERL_OPTIONS="+T 9"
```
Comment (henrik) [inline .travis.yml:11]: Oh, cool, didn't know about this. https://twitter.com/jlouis666/status/574685606703398913

**Comment by henrik** [comment]
Love it. Thank you so much for doing this :)

File: .travis.yml (hunk 1 above)
Comment (mindreframer) [inline .travis.yml:11]: it's a copy from https://github.com/elixir-lang/ecto/blob/master/.travis.yml ;) 

Actually, I did not know that this is for scheduler randomization... good to know!

**Comment by mindreframer** [comment]
@henrik thanks for writing about elixir, love your posts! Cheers!

**Comment by henrik** [comment]
@mindreframer :blush: Tests passed! https://travis-ci.org/henrik/ectoo

**Comment by mindreframer** [comment]
:+1:

**Pull request merged**